tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/pageobjects/Views.java

```diff
@@ -115,4 +119,18 @@ public SWTBotView showConsole() {
         return consoleView;
     }
 
+
+    /**
+     * Gets the view specified by the given id, if the view is open, otherwise returns null.
+     * @param viewId of view will get returned, if open
+     * @return view or null, if not found
+     */
+    public static IViewPart getViewById(String viewId) {
+        AtomicReference<IViewPart> view = new AtomicReference<>();
```

Comment:
Why not using SWTBot here? If I understand correctly this is not about opening a view, but finding a view right?
I would expect an iteration on current views provided by SWTBot till you find the needed one. Something similar is done somewhere else in our UI tests.

Reply:
Will be gone after #167